src/transformers/models/git/convert_git_to_pytorch.py

```diff
@@ -193,10 +194,27 @@ def prepare_img(model_name):
 
 
 def prepare_video():
-    from decord import VideoReader, cpu
+    def read_video_pyav(container, indices):
+        """
+        Decode the video with PyAV decoder.
 
-    # set seed for reproducability
-    np.random.seed(0)
+        Args:
+            container (`av.container.input.InputContainer`): PyAV container.
+            indices (`List[int]`): List of frame indices to decode.
+
+        Returns:
+            result (np.ndarray): np array of decoded frames of shape (num_frames, height, width, 3).
+        """
+        frames = []
+        container.seek(0)
+        start_index = indices[0]
+        end_index = indices[-1]
+        for i, frame in enumerate(container.decode(video=0)):
+            if i > end_index:
+                break
+            if i >= start_index and i in indices:
+                frames.append(frame)
```

Comment:
```suggestion
                frames.append(frame.to_rgb())
```
any reason we don't use this? 
or directly convert before stacking?

Reply:
Good question, we return the transformation at the end to directly transform the numpy data structure

```
return np.stack([x.to_ndarray(format="rgb24") for x in frames])
```

It's more efficient than converting the frames to rgb:
https://github.com/PyAV-Org/PyAV/blob/main/av/video/frame.pyx#L252
https://github.com/PyAV-Org/PyAV/blob/main/av/audio/frame.pyx#L168